### PR TITLE
Bug/r2fixes

### PR DESCRIPTION
**Pull request by kozmaz87**
This fixes couple of outstanding issues, updates the documentation and generally brings piece and prosperity to the developed World.

### CODE REVIEW

**Comment by nigelmegitt** [comment]
Documentation build generates a strange error:

```shell
$ python setup.py build_sphinx
running build_sphinx
creating /home/travis/build/ebu/ebu-tt-live-toolkit/docs/build/doctrees
creating /home/travis/build/ebu/ebu-tt-live-toolkit/docs/build/html
Running Sphinx v1.5.2
loading pickled environment... not yet created
building [mo]: all of 0 po files
building [html]: all source files
updating environment: 25 added, 0 changed, 0 removed
reading sources... [100%] websocket_carriage_mechanism                          
/home/travis/build/ebu/ebu-tt-live-toolkit/ebu_tt_live/bindings/__init__.py:docstring of ebu_tt_live.bindings.style_type.default_attrs:4: WARNING: Field list ends without a blank line; unexpected unindent.
/home/travis/build/ebu/ebu-tt-live-toolkit/ebu_tt_live/bindings/validation/presentation.py:docstring of ebu_tt_live.bindings.validation.presentation.StyledElementMixin._semantic_collect_applicable_styles:5: WARNING: Field list ends without a blank line; unexpected unindent.
```

**Comment by kozmaz87** [comment]
The documentation one is just a docstring indentation error. Finding and fixing

File: docs/source/nodes_and_carriage_mechanisms.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ Processing Nodes
 ----------------
 
    The processing nodes are the key components of this package and facilitate several different use-cases laid out
-   in the Node conformance section of EBU-3370. A node can be a producer or a consumer or both. This requirement
+   in the Node conformance section of the specification (EBBU Tech3370). A node can be a producer or a consumer or both. This requirement
```
Comment (nigelmegitt) [inline docs/source/nodes_and_carriage_mechanisms.rst]: s/EBBU/EBU

File: docs/source/timing_resolution.rst [hunk 2]
```diff
@@ -69,6 +69,6 @@ Sequence Timings
 ----------------
 
 The sequence timings are handled by the :py:class:`ebu_tt_live.documents.ebutt3.EBUTT3DocumentSequence` class.
-The document is insterted after having been validated into the sequence. The sequence looks at the computed begin and
-end times and detects collisions. If there are any the collisions are resolved by the logic starting in
-:py:func:`ebu_tt_live.documents.ebutt3.EBUTT3DocumentSequence._insert_or_discard`
\ No newline at end of file
+The document is insterted into the sequence after it is validated. The sequence looks at the computed begin and
```
Comment (nigelmegitt) [inline docs/source/timing_resolution.rst]: s/insterted/inserted

**Pull request merged**